### PR TITLE
KB migration: Create a patch

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds steps for creating a custom patch and applying it using Composer.

Specifically, it moves content from the existing [Create a Patch](https://support.magento.com/hc/en-us/articles/360005484154-Create-a-patch-for-a-Magento-2-Composer-installation-from-a-GitHub-commit) KB article from the Magento Knowledgebase (KB) to the DevDocs site.

I'll link to the staging build in the MAGEDOC issue.

Related to #1956.

## Affected documentation

- https://devdocs.magento.com/guides/v2.2/comp-mgr/patching.html#using-composer-to-install-patches

whatsnew
Added steps for [creating a patch from a git commit](https://devdocs.magento.com/guides/v2.2/comp-mgr/patching.html#custom-patches).

### CODE REVIEW

File: guides/v2.2/comp-mgr/patching.md [hunk 1]
```diff
@@ -28,7 +23,11 @@ Patch (or diff) files are text files that note:
 
 When the [patch][] program is run, this file is read in and the specified changes are made to the file(s).
 
-## Using the command line to apply a patch
+## Official patches
+
+You can find official Magento patches in the [Magento Security Center][]. Follow the instructions on the page to download the patch file, depending on your version and installation type.
+
+## Use the command line to apply a patch
```
Comment (keharper) [inline guides/v2.2/comp-mgr/patching.md]:
```suggestion
### Use the command line to apply an official patch
```

Comment (keharper) [reply guides/v2.2/comp-mgr/patching.md]: I can't make a comment near line 36, but the patch command should be placed in a bash code block.

Comment (jeff-matthews) [reply guides/v2.2/comp-mgr/patching.md]: @keharper, I think that's because you were looking at an outdated version of changes.

**Comment by keharper** [comment]
Re-reviewed, since Jeff noted that I was previously looking at an old commit. 

Double-plus approved.

**Comment by jeff-matthews** [comment]
Thanks @keharper! Adding @dobooth as well since he was the last one to edit this file and we just spoke about it.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**